backend/app/api/v1/projects.py

```
@@ -0,0 +1,291 @@
+        project_id, current_user, db,
+        required_roles=[UserRole.PROJECT_OWNER, UserRole.PROJECT_MANAGER, UserRole.TEAM_MEMBER, UserRole.VIEWER]
+    )
+    
+    members = project_member.get_by_project(db, project_id=project_id, skip=skip, limit=limit)
+    return members
+
+
+@router.post("/{project_id}/members", response_model=ProjectMember)
+def add_project_member(
+    *,
+    db: Session = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)